odp/Shapes/ShapesFunctions.py

```diff
@@ -90,7 +106,8 @@ def Upper_Half_Space(grid, dim, value):
             data += -grid.vs[i] + value
     return data
 
-def Union(shape1, shape2):
+
+def Union(shape1: np.ndarray, shape2: np.ndarray) -> np.ndarray:
```

Comment:
Perhaps use a reduce function to support any number of args

https://numpy.org/doc/stable/reference/generated/numpy.ufunc.reduce.html